app/components/ui/TwoColumnContentSection/TwoColumnContentSection.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { SanityImageSource } from "@sanity/image-url/lib/types/types.d";
+import BlockContent from "@sanity/block-content-to-react";
+import imageUrlBuilder from "@sanity/image-url";
+import styles from "./TwoColumnContentSection.module.scss";
+import { client } from "../../../sanity";
+
+const builder = imageUrlBuilder(client);
+
+const BlockRenderer = (props: { node: any; children: any }) => {
+  const { node, children } = props;
+  const { style = "normal" } = node;
+  if (style === "h1") {
```

Comment:
I'd say don't worry about this right now, but I wonder if we should enforce the headers being h2 to minimize accessibility errors for whomever is adding content. They should likely never be h1, since the page header will be the h1 and most likely would never need to be smaller than h2, given that they'll probably be placed directly under the page header.

Reply:
ohh i like that a lot. I also wonder if we can enforce that in sanity studio.